### ISSUE TITLE
No support for *.cjs and *.mjs file extensions

### ISSUE THREAD

**Issue by petervmeijgaard** [serverless-stack/open-next#5]
First of all, thank you so much for the hard work on this library. This will help out so many companies in the future :)

I tried using the CLI locally, but ran into an error:
```
Error: next.config.js not found. Please make sure you are running this command inside a Next.js app.
```

However, I'm using Node 18 with the `.cjs` and `.mjs` file extensions to separate ESM and CJS file types. `open-next` only seems to pick up on the `.js` file-extension. ([build.ts#L14](https://github.com/serverless-stack/open-next/blob/main/cli/src/build.ts#L14))

Could you add support for both the `.cjs` and `.mjs` file extensions and types?

**Comment by petervmeijgaard** [comment]
I tried it out with this project, in case you're interested and would like to reproduce the error:
https://github.com/petervmeijgaard/shorty

**Comment by fwang** [comment]
Thanks @petervmeijgaard. Will take a look.

**Comment by fwang** [comment]
Fixed in v0.5.1
